ArduPlane/mode_cruise.cpp

```diff
@@ -67,7 +67,7 @@ void ModeCruise::navigate()
         plane.channel_roll->get_control_in() == 0 &&
         plane.rudder_input() == 0 &&
         plane.gps.status() >= AP_GPS::GPS_OK_FIX_2D &&
```

Comment:
We don't seem to be consistent about checking for 3D vs 2D fix.

Reply:
true, maybe in another PR